akka-docs/src/main/paradox/serialization-jackson.md

```diff
@@ -361,6 +365,63 @@ binding, but it should still be possible to deserialize old data with Jackson.
 
 It's a list of class names or prefixes of class names.
 
+## Rolling upgrades
+
+When doing a rolling upgrade, for a period of time there are two different binaries running in production. If the schema
```

Comment:
I prefer "rolling update", but we have use some mix of both anyway

Reply:
raised https://github.com/akka/akka/pull/29523